packages/protobuf/src/google/protobuf/wrappers_pb.ts

```diff
@@ -79,8 +79,8 @@ export class DoubleValue extends Message<DoubleValue> {
   ]);
 
   static readonly fieldWrapper = {
-    wrapField(value: number | DoubleValue): DoubleValue {
-      return value instanceof DoubleValue ? value : new DoubleValue({value});
+    wrapField(value: number): DoubleValue {
+      return new DoubleValue({value});
     },
```

Comment:
This simplifies the generated code by making the internal `function wrapField` idempotent instead of this FieldWrapper. I don't like changing this, but it the generated code does look quite a bit nicer...